field-agent-ui/public/index.html

```diff
@@ -4,10 +4,15 @@
   <meta charset="utf-8" />
   <title>Field Agent</title>
   <link rel="stylesheet" type="text/css" href="../src/styles/reset.css">
+  <link rel="stylesheet" href="/bootstrap-journal.min.css">
+  <link rel="stylesheet"
+        href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
   <link rel="stylesheet" type="text/css" href="../src/styles/base.css">
   <link rel="stylesheet" type="text/css" href="../src/styles/style.css">
```

Comment:
These CSS files should be moved into the `public` folder. If you want to leave them in the `src` folder, you should import them into your `index.js` file instead of linking them into the `index.html` file.